Jellyfin.Api/Helpers/StreamingHelpers.cs

```diff
@@ -508,17 +508,18 @@ private static string GetOutputFilePath(StreamState state, string outputFileExte
 
         private static void ApplyDeviceProfileSettings(StreamState state, IDlnaManager dlnaManager, IDeviceManager deviceManager, HttpRequest request, string? deviceProfileId, bool? @static)
         {
-            var headers = request.Headers;
-
             if (!string.IsNullOrWhiteSpace(deviceProfileId))
             {
-                state.DeviceProfile = dlnaManager.GetProfile(deviceProfileId);
-            }
-            else if (!string.IsNullOrWhiteSpace(deviceProfileId))
-            {
-                var caps = deviceManager.GetCapabilities(deviceProfileId);
+                if (state.DeviceProfile == null)
```

Comment:
You sure? I would expect `deviceProfileId` to take precedence tbh

Reply:
Good catch - don't know where my head was at!